app/assets/stylesheets/home.scss

```diff
@@ -21,11 +20,25 @@
   }
 }
 
-div.module.text-center {
-  border-radius: 50px;
+.pie-size {
+  background: asset-url("path.jpg") no-repeat center center;
+  height: 300px;
+  position: relative;
 }
 
-.learn {
-  background-color: #d5ffea;
-  border-radius: 50px;
+.vertical-center {
+  position: absolute;
+  top: 25%;
+
+  input {
+    background-color: rgba(0,0,0, 0.18);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.18)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space